decidim-debates/spec/commands/decidim/debates/create_debate_spec.rb

```diff
@@ -105,6 +108,26 @@
     end
   end
 
+  context "when everything is ok with attachments" do
```

Comment:
Could you add a spec with what happens what if you upload a document that is not allowed? for example a ODT file ?